clients-simple/src/main/java/uk/gov/gchq/palisade/clients/simpleclient/web/ResourceClient.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.palisade.clients.simpleclient.web;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import uk.gov.gchq.palisade.resource.request.AddResourceRequest;
```

Comment:
This request will be moved from common to services when current work has been completed. I would recommend copying this request over to clients

Reply:
The AddResourceRequest should to be copied across to this repo.

Reply:
Sorry, I missed this one the first time around.  I am going to push back on this idea.  This is perfectly good idea, but the AddResourceRequest is one of a set of five classes in the common repo that all extend the the Request class.  I would argue that moving one class in the common repo out to another should be treated as part of a new change request that looks as the whole common repo.   The current exercise was to pull the SimpleClient (and related dependencies) out of the example repo and put it in the client repo.

Reply:
The only reason I mentioned this is because the work being done for 390 and 639 are moving these requests out of common and will therefore mean that this will need to be done before this is merged.

Reply:
Ah, so the code will not necessarily compile or run until after the other changes.  Ok we wait and make the changes that we need to after the code is checked in for 390 and 639.